src/server/locale/US/index.js

```diff
@@ -9,6 +9,7 @@ export default offerType => {
         case 'GPLNQ':
         case 'GPLNQ_RANGE':
             return gpl;
+        case 'LT_NO_AMMT':
```

Comment:
```suggestion
        case 'LT_NO_AMNT':
```

In this same file we can also remove the cases 
` case 'LT_NQ_MIN':`, `case 'LT_NQ_MAX':`, `case 'LT_NQEZ_MIN':`, and `case 'LT_NQEZ_MAX':`.